pkg/policy/consumer.go

```diff
@@ -259,25 +280,29 @@ func (c *Consumable) AllowConsumerLocked(cache *ConsumableCache, id NumericIdent
 // consumers map and the given consumable to the given consumer's consumers map.
 // Must be called with Consumable mutex Locked.
 func (c *Consumable) AllowConsumerAndReverseLocked(cache *ConsumableCache, id NumericIdentity) {
-	log.Debugf("Allowing direction %d -> %d", id, c.ID)
+	scopedLog := log.WithFields(log.Fields{
+		logfields.Identity + ".from": c.ID,
```

Comment:
Original message specified opposite direction: `id -> c.ID`.

Reply:
Oops! so it did. Fixed.